packages/grafana-data/src/themes/colorManipulator.test.ts

```diff
@@ -272,7 +272,7 @@ describe('utils/colorManipulator', () => {
     });
 
     it('converts an hsl color to an hsla color with the value provided', () => {
-      expect(alpha('hsl(0, 100%, 50%)', 0.1)).toEqual('hsla(0, 100%, 50%, 0.1)');
+      expect(alpha('hsl(0, 100%, 50%)', 0.1)).toEqual('hsl(0, 100%, 50%, 0.1)');
```

Comment:
This also looks like it will break?

Reply:
same story as rgba(): https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/hsl()